R/pkg-arrow.R

```diff
@@ -0,0 +1,36 @@
+#' Create a arrow Table from a Polars object
+#'
+#' @param x [A Polars DataFrame][DataFrame_class]
+#' @param ... Ignored
+#' @rdname S3_as_arrow_table
+#' @examples
+#' library(arrow)
+#'
+#' pl_df = as_polars_df(mtcars)
+#' as_arrow_table(pl_df)
+# exported in zzz.R
```

Comment:
```suggestion
# exported in zzz.R: this generic belongs to `arrow` and we only export the method if 
# `arrow` is already loaded when we load `polars`
```
just a note for future us

Reply:
There are similar things elsewhere, so if you are interested, please change them all by another PR.